features/basic_replication.py

```diff
@@ -0,0 +1,49 @@
+import psycopg2 as pg
+from time import sleep
+
+from lettuce import world, steps
+
+
+@steps
+class BasicReplicationSteps(object):
+
+    def __init__(self, environ):
+        self.env = environ
+
+    def start_patroni(self, step, name):
+        '''I start (\w+)'''
+        return world.pctl.start(name)
+
+    def stop_patroni(self, step, name):
+        '''I shut down (\w+)'''
+        return world.pctl.stop(name)
+
+    def kill_patroni(self, step, name):
+        '''I kill (\w+)'''
+        return world.pctl.stop(name, kill=True)
+
+    def add_table(self, step, table_name, pg_name):
+        '''I add the table (\w+) to (\w+)'''
+        # parse the configuration file and get the port
+        try:
+            world.pctl.query(pg_name, "CREATE TABLE {0}()".format(table_name))
+        except pg.Error as e:
+            assert False, "Error creating table {0} on {1}: {2}".format(table_name, pg_name, e)
+
+    def table_is_present_on(self, step, table_name, pg_name, max_replication_delay):
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/cyberdemn/patroni/file/2114827816/issues/source?bid=3127594&fileBranchId=3144623#l33)